### ISSUE TITLE
Receiving TypeError in graffiti 0.1.3

### ISSUE THREAD

**Issue by marcioj** [jayphelps/graffiti#2]
I have cloned the https://github.com/jayphelps/graffiti-example, since it looks the easy way to start with graffiti. Without change anything I run `npm run build` go to the `dist` folder and open the `index.html` file in the browser. When the page loads, not is display and in the console I can see an error: `TypeError: Cannot read property 'apply' of undefined`.

Using the previous version 0.1.2 works fine.

Like I described [here](https://github.com/jayphelps/graffiti-runtime/commit/adf6767947d748bae64cc3585a43c7504ee98e07#commitcomment-12242775), this error is threw because `Object.getPrototypeOf(_obj)` returns an object which doesn't have a `setAttribute` property.

These are the related files, just the example component that comes in `graffiti-example`:

_my-counter.js_

``` js
import { registerElement, reflectToAttribute } from 'graffiti';

export default
@registerElement()
class MyCounterComponent extends HTMLElement {
  @reflectToAttribute()
  counter = 0;

  events = {
    increment() {
      this.counter++;
    }
  }
}
```

_my-counter.hbs_

``` hbs
<my-counter>
  <div id="label"><content></content></div>
  Value: <span id="counter-value">{{counter}}</span><br>
  <button {{on "click" "increment"}}>Increment</button>
</my-counter>
```

**Comment by jayphelps** [comment]
Figured out the issue. It had worked previously because babel _used_ to dynamically bind `super.setAttribute`, but this was actually incorrect; the spec says it should be statically bound. https://github.com/babel/babel/issues/943 I was still relying on this incorrect behavior because my local install of babel was still old and didn't cause this.

Thanks reporting!

**Comment by jayphelps** [comment]
Released on bower as [v0.1.4](https://github.com/jayphelps/graffiti-runtime/releases/tag/v0.1.4)

**Comment by marcioj** [comment]
It worked fine here. Thank you for the fix!
